test/presenter/Chart.spec.js

```diff
@@ -684,9 +697,9 @@ function (
 
 
         describe('HTML smoke tests', function() {
```

Comment:
These updates are just to include the new `axis_label--x` and `axis_label--y` classes